client-js/app/index.html

```diff
@@ -33,7 +33,8 @@
   <link rel="stylesheet" href="/styles/main.css">
   <script src="/bower_components/webcomponentsjs/webcomponents-lite.js"></script>
   <link rel="import" href="/elements/elements.html">
-  <!-- PROD_MODE -->
+
+  <!-- Websocket url for dev mode -->
```

Comment:
Let's go over once again what this is doing. The comment here appears in index.html, which is served AS IS* in prod mode. In dev mode, this comment gets _replaced_ by a script which overrides the url where we should connect to the api backend. Originally I wrote PROD_MODE because if you look at the source of this file in PROD_MODE it would then not look out of place, but of course this does not then convey any information about why the comment is there at all.

The change here is meant to have the comment describe why it is there, e.g. to be replaced by the dev mode websocket url when running in dev mode. As it happens, we minify the html before using it in prod mode, so comments get stripped and it won't happen that someone looking at the source in prod mode will see the comment either way, so my original reasoning for writing PROD_MODE is kind of moot.

In the end, though, this is just a placeholder token. We can write here _whatever we want_. If you tell me what you think would be least confusing to a new reader of the code, I'd be happy to use that. It just has to match between here and the spot in the gulpfile where we do the replacement.